scss/forms/_floating-labels.scss

```diff
@@ -58,7 +58,9 @@
   > .form-control-plaintext,
   > .form-select {
     ~ label {
-      transform: $form-floating-label-transform;
```

Comment:
This removes the animation when clicking in an input or textarea with a floating label.

Test the change at https://deploy-preview-40735--twbs-bootstrap.netlify.app/docs/5.3/forms/floating-labels/.

Reply:
I will go through the code again and review the changes. Thanks for pointing this out.